flyteidl/scripts/add_pyo3_subclass.sh

```diff
@@ -0,0 +1,12 @@
+sed '/pub struct /i\
+#[pyo3(subclass)]\
```

Comment:
Why this hacky workaround?
In prost builder configuration, protobuf compiler we used in `gen.rs` can't specify PyO3's `subclass` for **only** Rust `struct`. Because , `enum` is a subset of `type`, it's non-trivial to distinguish between these while adding attributes.

Reply:
So we add this auxiliary scripts